lib/core/nuxt.js

```diff
@@ -142,7 +142,7 @@ export default class Nuxt {
     this.readyMessage = null
   }
 
-  listen(port = 3000, host = 'localhost', socket = null) {
+  listen(port = null, host = null, socket = null) {
```

Comment:
Just a small nit: Do we have to provide default values here?

Reply:
I don't think so, will remove them, good catch @manniL